app/assets/stylesheets/responsive.scss

```diff
@@ -1,2 +1 @@
-@import "bootstrap-responsive";
 @import "bootstrap-responsive-overrides";
```

Comment:
Do you want to do that as part of this PR, or later? I'm good either way, just wanting to confirm if I should merge this or not.